app/views/courses/show.html.erb

```diff
@@ -6,10 +6,7 @@
 <% end %>
 
 <p>
-  <b>Status:</b>
-  <span class="label <%= success_or_warning_label(@course.status, 'Active') %>">
-    <%= @course.status %>
-  </span>
+  <b>Status:</b> <%= success_or_warning_label(@course.status, 'Active') %>
```

Comment:
It looks like this would just have the label/class ? Wouldn't we still need `<%= @course.status %>`

And if you wanted to throw in a test on this....